### PR TITLE
Stable links

### PR DESCRIPTION
**Pull request by jozzi**
Operators can be linked with short links as following:

/operator/[operator-id] -> points to default channel and latest version there
/operator/[channel]/[operator-id] -> points to channel and gets latest version for it
/operator/[channel]/[operator-name] /> same as current links

### CODE REVIEW

File: server/services/operatorsService.js [hunk 1]
```diff
@@ -1,37 +1,159 @@
...
+const fetchOperator = (serverRequest, serverResponse) => {
+  const operatorName = _.get(serverRequest, 'query.name', '');
+  const channelName = _.get(serverRequest, 'query.channel');
+
+  let operatorId = operatorName;
+
+  // use method only if there is dot 
+  if (operatorName.indexOf('.') > -1) {
+    operatorId = generateIdFromVersionedName(operatorName);
+  }
```
[the remark this answers is not in the document]
Comment (jozzi) [reply server/services/operatorsService.js]: It does not work in that case.
`const generateIdFromVersionedName = name => name.slice(0, name.indexOf('.'));`
index of return -1 if there is no '.' and slice accepts negative index - it substracts it from the end, so effectively it drops last character.

Did it this way and it did not worked so added the check :)

**Comment by jozzi** [comment]
Added tiny change to the PR - there was forgotten full name with version in anchor href of Tile view

**Comment by galletti94** [comment]
@jozzi can you make this PR against the dev branch instead of master?

**Comment by jozzi** [comment]
Closed and opened new PR against dev branch

**Pull request closed without merging**